cmd/bee/cmd/cmd.go

```diff
@@ -190,4 +194,8 @@ func (c *command) setAllFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool(optionNameGatewayMode, false, "disable a set of sensitive features in the api")
 	cmd.Flags().Bool(optionNameClefSignerEnable, false, "enable clef signer")
 	cmd.Flags().String(optionNameClefSignerEndpoint, "", "clef signer endpoint")
+	cmd.Flags().String(optionNameSwapEndpoint, "http://localhost:8545", "swap ethereum blockchain endpoint")
+	cmd.Flags().String(optionNameSwapFactoryAddress, "", "swap factory address")
+	cmd.Flags().Uint64(optionNameSwapInitialDeposit, 0, "initial deposit if deploying a new chequebook")
+	cmd.Flags().Bool(optionNameSwapEnable, false, "enable swap")
```

Comment:
As we spoke about. Don't remove this flag, but set default to `true` whenever possible from devops (@vandot) side.

https://github.com/ethersphere/bee/pull/703/files#r491862821